rpc_jobs/rpc_aio.yml

```diff
@@ -346,7 +343,6 @@
                 environment_vars = [
                   "DEPLOY_HAPROXY=yes",
                   "DEPLOY_AIO=no",
-                  "DEPLOY_MAAS=no",
```

Comment:
DEPLOY_MAAS is already an environment variable because it is a parameter, so it doesn't need to be in this array. I don't think this array is needed anyway. Why pass env vars around, why not set them directly?